src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -15,11 +15,14 @@
 		<PublishTrimmed>True</PublishTrimmed>
 		<TrimMode>partial</TrimMode>
 	</PropertyGroup>
+	<PropertyGroup Condition="'$(TargetFramework)'=='net8.0-ios'">
```

Comment:
this section does not need to be commited

Reply:
Oups nice catch might need to make it cleaner, without this, it's impossible to build for iOS unless the Visual Studio is configured in a specific way, easier to have manual with the automatic option set.